queries/editorconfig/highlights.scm

```diff
@@ -0,0 +1,51 @@
+(comment) @comment @spell
+
+; consistency with toml and ini
+(section
+  (section_name) @type)
+
+(character_choice
+  (character) @constant)
+
+(character_range
+  start: (character) @constant
+  end: (character) @constant)
+
+[
+  "["
+  "]"
+  "{"
+  "}"
+] @punctuation.bracket
+
+[
+  ","
+  (path_separator)
+] @punctuation.delimiter
+
+[
+  "-"
+  "="
+  ".."
+  (negation)
+  (wildcard_characters)
+  (wildcard_any_characters)
+  (wildcard_single_character)
+] @operator
```

Comment:
```suggestion
[
  "-"
  "="
  ".."
  (negation)
] @operator

[
  (wildcard_characters)
  (wildcard_any_characters)
  (wildcard_single_character)
] @character.special
```

Reply:
Yeah, that would make more sense.

Reply:
wait, here `..` doesn't mean 'parent directory' but is used to indicate an integer range: `{1..10}`.